AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -449,6 +449,11 @@ - (ASCellNode *)nodeForItemAtIndexPath:(NSIndexPath *)indexPath
   return [_dataController nodeAtIndexPath:indexPath];
 }
 
+- (ASCellNode *)supplementaryNodeOfKind:(NSString *)kind atIndexPath:(NSIndexPath *)indexPath
```

Comment:
Rename to `supplementaryNodeForElementKind:atIndexPath:` to align with UIKit naming semantics.